### DISCUSSION TITLE
Multiple authors

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#8134] [Ideas] [answered]
> Created by: PlutoMars (Limey.pluto@gmail.com) on 2015/03/20 09:37:19 +0000
> Votes at time of UserVoice import: 14

-----------------------------------

It would be very useful if multiple authors could be assigned to a single article. Especially 2-3 editors cooperating closely on a single article, without giving entire groups access, or having to create user groups for 2-3 people.

**Comment by angrybrad** [comment]
> Posted by Anonymous (jmenberg@web.de) on 2015/07/11 08:47:41 +0000
-----------------------------------

Yes please! That would be perfect. As multi-author blogging / collaborative writing gets generally more and more important, this feature seems quite necessary. Thanks for consideration!

**Comment by angrybrad** [comment]
> Posted by PlutoMars (Limey.pluto@gmail.com) on 2015/03/20 09:37:19 +0000
-----------------------------------

In ExpressionEngine this is possible... (one of the few things I still like about it)

**Comment by timkelty** [comment]
Along similar lines, it would be nice if entries were allowed to have NO author.

I've wanted this specifically when dealing with imported entries with no author.

Currently, I end up having to import to a "default" user, and then check if an entry is authored by that user before I display the author attribution.

Additionally, when using something like the [Guest Entries plugin](https://github.com/pixelandtonic/GuestEntries), you wouldn't need to assign a default author.

**Comment by narration-sd** [comment]
@timkelty, think it would be wiser to keep that default user.

There are a lot of things in Craft that depend on identity and privileges; would no doubt be hard to circumvent them.  Thus sensilbly you are back to a default user/author, and in more control if you provide it. 

I have used the admin as author, of course, but later changed from this and don't recommend it, for reasons you can no doubt see...

**Comment by timkelty** [comment]
@narration-sd I totally get what you're saying, but not sure I agree. Certainly not suggesting it would be a trivial feature to implement by any means.

> There are a lot of things in Craft that depend on identity and privileges; would no doubt be hard to circumvent them.

Very true, however we're mostly just talking about validation changes to `EntryModel`. Other elements (e.g. assets, categories) don't even have authors at all.

Craft's granular permissions are actually why I'd like to see special handling for a "no author" scenario. Consider the following:

- A section of `reviews` entries
- A member group `siteMembers` that has permission to:
  - Edit & Publish `reviews`
  - ...But _NOT_ `Edit other authors’ entries` (i.e. they can only edit their own `reviews`)
- A set of import/programmatically created `reviews`, some with no author, so were assigned a "default user".

At this point I'd like `siteMembers` to be able to edit `reviews` that don't have a "real" author, but can't because their permissions don't allow it.

**Comment by bzin** [comment]
Was wondering about this feature request  @brandonkelly. Are you guys shipping this soon on a 3.1 version or in a later? :)

**Comment by brandonkelly** [comment]
@bzin No current plans to implement multi-author entries right now.

**Comment by boboldehampsink** [comment]
I could use this for a lot of my clients. I use it to set permissions on an per entry-basis

**Comment by furioursus** [comment]
This would be amazing for @cityofoakland, as we often have staff that our digital leads only want accessing *one* single entry in a section on the site.

**Comment by watershed** [comment]
Just adding my +1 to this feature request. 

One of the _very_ few areas where I find Craft will not bend to my will ;)

I see above that it's added to the planned scope of Craft 3.6, which is great news.

**Comment by amimpact** [comment]
Please add this feature as soon as possible! We have a lot of use cases where we need this.

**Reply by TUSAB-web** [reply]
Agreed. I like having the permissions set so that authors can create and delete and edit their own stuff, but not other authors’ stuff. However, sometimes they need to. Then I have to give them the ability to edit everything in that section. If I could simply add multiple authors to entries, it would be easier to make sure authors are only able to edit certain items.

**Comment by adrienne** [comment]
I believe (@brandonkelly can confirm) that this is on the roadmap for Craft 4?

**Reply by brandonkelly** [reply]
Yep: https://craftcms.com/roadmap

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.0.0-alpha.4 was just released with multi-author support 🎉 (#12380)
